shopfloor/actions/data.py

```diff
@@ -64,8 +65,10 @@ def package(self, record, picking=None, with_packaging=False, **kw):
         data = self._jsonify(record, parser, **kw)
         # handle special cases
         if data and picking:
-            # TODO: exclude canceled and done?
-            lines = picking.move_line_ids.filtered(lambda l: l.package_id == record)
+            lines = picking.move_line_ids.filtered(
+                lambda l: l.result_package_id == record
+                and l.state in ["partially_available", "assigned", "done"]
```

Comment:
@JuMiSanAr this is the change the required the tests update, not the reception scenario introduction

Reply:
Not sure I understand what you mean here.

Reply:
I mean that the fix for the tests has been committed into the reception but it has nothing to deal with it